### ISSUE TITLE
Site connection error

### ISSUE THREAD

**Issue by dequadelisonz** [SIMS-IOT-Devices/HTTP-ESP-IDF#1]
Hello I get the following error imediatly after connecting to wifi:

I (552) wifi:Total power save buffer number: 16
I (552) wifi:Init max length of beacon: 752/752
I (552) wifi:Init max length of beacon: 752/752
WIFI was initiated ...........

E (2552) esp-tls: couldn't get hostname for :worldclockapi.com: getaddrinfo() returns 202, addrinfo=0x0
E (2552) TRANSPORT_BASE: Failed to open a new connection: 32769
E (2552) HTTP_CLIENT: Connection failed, sock < 0

I'm using ESP32S3 devkit v1.1

**Comment by avehbiaslanturk** [comment]
Yes it is the same,
Did you solve it?

**Comment by dequadelisonz** [comment]
Yes, didn't exactly understood how... But suddenly started to work. Maybe because I changed max available stack limit but not sure. Anyway thx.
